phpBB/phpbb/console/command/thumbnail/delete.php

```diff
@@ -38,16 +48,16 @@ class delete extends \phpbb\console\command\command
 	/**
 	* Constructor
 	*
-	* @param \phpbb\config\config $config The config
-	* @param \phpbb\user $user The user object (used to get language information)
-	* @param \phpbb\db\driver\driver_interface $db Database connection
-	* @param string $phpbb_root_path Root path
+	* @param user $user The user object (used to get language information)
+	* @param driver_interface $db Database connection
+	* @param language $language Language
+	* @param storage $storage Storage
 	*/
-	public function __construct(\phpbb\config\config $config, \phpbb\user $user, \phpbb\db\driver\driver_interface $db, $phpbb_root_path)
```

Comment:
I would recommend ordering these a-z as none of them are optional

Reply:
I have undone this to follow the format of the rest of the commands.
First goes user (that is from the parent class) and then the rest